packages/frontend/src/components/Home/OnboardingPanel/index.tsx

```diff
@@ -42,45 +30,45 @@ const onboardingSteps = [
 ];
 
 const OnboardingPanel: FC<Props> = ({ projectUuid, userName }) => {
-    const { track } = useTracking();
     return (
-        <OnboardingPanelWrapper>
-            <Title>
+        <Stack justify="flex-start" spacing="xs" mt="xl" pt="xl">
+            <Title order={3}>
                 {`Welcome${userName ? ', ' + userName : ' to Lightdash'}! 👋`}
             </Title>
-            <Intro>
+            <Text color="gray.7">
                 You&apos;re ready to start exploring. Here&apos;s what you can
                 do with Lightdash:
-            </Intro>
-            <CardWrapper>
-                <StepsWrapper>
-                    {onboardingSteps.map((step, item) => (
-                        <StepContainer key={step.title}>
-                            {step.image}
-                            <StepTitle>{step.title}</StepTitle>
-                            <StepDescription>
+            </Text>
+            <Paper withBorder={true} w={900} p="xl" mt="lg">
```

Comment:
withBorder={true} can be  simpler, just withBorder
w={900} is hardcoded, should probably be "100%" or "inherit"